### PR TITLE
Refactor housekeeping and update file extensions 

### LINKED ISSUE #9: Adapt README
To Markdown, the name of Raku and whatever else seems obsolete.

### PR DESCRIPTION
**Pull request by camstuart**
General housekeeping maintenance 

- Updated file extensions to suit current Raku convention
- Reformat README to pod style, but still compatible with GitHub rendering
- Add `LICENSE` file
- Update `META6.json`

Resolves #9 & #10

### CODE REVIEW

**Comment by camstuart** [comment]
Any chance we can get this merged @JJ ? I'm also very interested to hear your other thoughts on this module. Once tidied up, I need to get this republished. It's failing on install for the refurbishment work on Raku/examples

File: README.pod [hunk 1]
```diff
@@ -0,0 +1,34 @@
+=encoding utf-8
+
```
Comment (JJ) [inline README.pod:2]: Don't really get this in Pod format; but it does not really matter. It's legible by GitHub, so I guess it's as good as markdown or anything like that.

File: t/mime.rakutest [hunk 2]
```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env raku
+
```
Comment (JJ) [inline t/mime.rakutest:2]: To tell you the truth, this is the first time I've seen this extension and I'm not totally sure it's canon. That might explain why tests have failed...

**Review by JJ — APPROVED** [review]
Thanks a lot and sorry for the delay. I don't totally share some choices, but I don't think they're a major problem.

**Comment by JJ** [comment]
> Any chance we can get this merged @JJ ? I'm also very interested to hear your other thoughts on this module. Once tidied up, I need to get this republished. It's failing on install for the refurbishment work on Raku/examples

We definitely need to get it going. I'll have a look at the errors. They're simply using panda, so we'll have to get CI back on track.

File: t/mime.rakutest (hunk 2 above)
Comment (camstuart) [reply t/mime.rakutest:2]: It's certainly the longest file extension hats for sure! I got the naming from here:

https://github.com/Raku/problem-solving/blob/master/solutions/language/Path-to-Raku.md#extensions

But happy to hear if there has / is a change.

Comment (JJ) [reply t/mime.rakutest:2]: Well, there you go. Tests worked; they failed because they were trying to use panda...

**Pull request merged**